components/camel-rabbitmq/src/main/java/org/apache/camel/component/rabbitmq/RabbitMQConnectionFactorySupport.java

```diff
@@ -0,0 +1,51 @@
+package org.apache.camel.component.rabbitmq;
+
+import java.security.KeyManagementException;
+import java.security.NoSuchAlgorithmException;
+
+import com.rabbitmq.client.ConnectionFactory;
+
+/**
+ * @author Milos Milivojevic | mmilivojevic@deployinc.com
```

Comment:
We do not include author tags in our javadoc at Apache Camel. Do you mind removing that?